### ISSUE TITLE
FlexboxLayout crashes app on iOS

### ISSUE THREAD

**Issue by bfv** [NativeScript/NativeScript#3737]
## Please, provide the details below:
The FlexboxLayout works perfectly on Android, the same code however crashes on iOS. 

### Did you verify this is a real problem by searching [Stack Overflow]
StackOverflow & a google Search didn't provide a solution 

### Tell us about the problem
The following code doesn't work on iOS:
```
<FlexboxLayout class="bottom-navbar" justifyContent="space-around">
    <ng-content></ng-content>
</FlexboxLayout>
```
The code where it goes wrong is in repo: 
[github.com/bfv/sailert](https://github.com/bfv/sailert/commit/d4b1b6532f4ca3e8a40e503ec7323c401de4cb05)
in 
`app\modules\bottomnavbar\components\bottom-navbar\bottom-navbar.component.html`

The app crashes with: 

```
CONSOLE LOG file:///app/tns_modules/@angular/core/bundles/core.umd.js:111:20: Angular is running in the development mode. Call enableProdMode() to enable the production mode.
1   0x10ae677cb NativeScript::FFICallback<NativeScript::ObjCMethodCallback>::ffiClosureCallback(ffi_cif*, void*, void**, void*)
2   0x10b53c1ee ffi_closure_unix64_inner
3   0x10b53cb22 ffi_closure_unix64
4   0x10f10fbf8 -[CALayer layoutSublayers]
5   0x10f103440 CA::Layer::layout_if_needed(CA::Transaction*)
6   0x10f1032be CA::Layer::layout_and_display_if_needed(CA::Transaction*)
7   0x10f091318 CA::Context::commit_transaction(CA::Transaction*)
8   0x10f0be3ff CA::Transaction::commit()
9   0x10f0bed6f CA::Transaction::observer_callback(__CFRunLoopObserver*, unsigned long, void*)
10  0x10ebba267 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__
11  0x10ebba1d7 __CFRunLoopDoObservers
12  0x10eb9ef8e __CFRunLoopRun
13  0x10eb9e884 CFRunLoopRunSpecific
14  0x112396a6f GSEventRunModal
15  0x10c0cac68 UIApplicationMain
16  0x10b53c97d ffi_call_unix64
17  0x12c4b08e0
file:///app/tns_modules/tns-core-modules/ui/core/view.js:96:124: JS ERROR Error: onMeasure() did not set the measured dimension by calling setMeasuredDimension() ProxyViewContainer(286)
Mar  3 13:06:50 10 com.apple.CoreSimulator.SimDevice.A98C6961-EB84-4078-A7AB-C3CF7FD903B6.launchd_sim[1197] (UIKitApplication:io.bfv.sailert[0xc964][1559]): Service exited due to Segmentation fault: 11
```

### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
```
┌──────────────────┬─────────────────┬────────────────┬─────────────┐
│ Component        │ Current version │ Latest version │ Information │
│ nativescript     │ 2.5.2           │ 2.5.2          │ Up to date  │
│ tns-core-modules │ 2.5.1           │ 2.5.1          │ Up to date  │
│ tns-android      │ 2.5.0           │ 2.5.0          │ Up to date  │
│ tns-ios          │ 2.5.0           │ 2.5.0          │ Up to date  │
└──────────────────┴─────────────────┴────────────────┴─────────────┘
```
### Please tell us how to recreate the issue in as much detail as possible.
Simple clone the mentioned above (the link contains the correct commit) and run the code.

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
Since this is an {N}/ng2 project I feel it's better to use the exact code as provided.

PS. It isn't entirely clear whether it's related to ng2 or a generic problem.

**Comment by NickIliev** [comment]
Hi @bfv and thanks for reporting this one!
The issue is successfully reproduced with any Angular related elements wrapped in FlexboxLayout.
Using any other layouts can be used as a workaround.

Test app demonstrating the issue can be found [here](https://github.com/NickIliev/NativeScript-Issues-2017/tree/master/NativeScript/issue_3737)

Logged error:
```
JS ERROR Error: onMeasure() did not set the measured dimension by calling set
MeasuredDimension() ProxyViewContainer(6)
```

**Comment by vchimev** [comment]
Hey @bfv,

This issue has been addressed in **tns-core-modules@2.5.1**. Related to: https://github.com/NativeScript/NativeScript/pull/3675.

**Comment by bfv** [comment]
@vchimev Thanks!
